src/test/java/org/hmhb/authentication/DefaultAuthenticationServiceTest.java

```diff
@@ -64,7 +64,7 @@ private GoogleOauthAccessRequestInfo prepGoogleCall() throws Exception {
                 gPlusProfile
         );
 
-        PublicConfig publicConfig = new PublicConfig(CLIENT_ID, null);
+        PublicConfig publicConfig = new PublicConfig(CLIENT_ID, null, 1, 2, 3, 4, 5);
```

Comment:
Notice how the details of this object's constructor spill out all throughout the tests.

I should probably write a Builder for the config classes or something, but not in this PR.